data/map.txt

```diff
@@ -29506,20 +29506,20 @@ planet "Tebuteb's Table"
 
 planet "Tefkar Ret"
 	attributes korath station
-	landscape land/station0
+	landscape land/station12
```

Comment:
The description says that this station is run down and abandoned, but station12 doesn't look very run down compared to station0.